buildtest/cli/__init__.py

```diff
@@ -1295,3 +1331,24 @@ def cdash_menu(subparsers):
     upload.add_argument(
         "-o", "--open", action="store_true", help="Open CDASH report in browser"
     )
+
+
+def add_hidden_parser(subparser, hidden_parser):
+    """This method adds a help message to certain parsers
```

Comment:
Change doctoring to 

This method will add parser for hidden commands that can be shown when using --help-all